### PR TITLE
css: improve contentcontrol datepicker

### PR DESCRIPTION
**Pull request by Rash419**
Signed-off-by: Rash419 <rashesh.padia@collabora.com>
Change-Id: Ie03578e47e123b6100fa361005a148a50bffa6d6

* Target version: master 

### Summary
Before:
image
After:
image

### CODE REVIEW

**Comment by Rash419** [comment]
@pedropintosilva decided to override the css rules because it gives me more control on how date picker looks

File: browser/css/cool.css [hunk 1]
```diff
@@ -900,3 +900,54 @@ nav.spreadsheet-color-indicator ~ #sidebar-dock-wrapper {
...
+#datepicker .ui-state-default {
+	background: #f7f7f7 ;
+	font-weight: normal;
+	color: black;
+	border: 1px solid #fff;
+}
+
+#datepicker .ui-widget-content {
+	background: #fff;
+}
```
Comment (pedropintosilva) [inline browser/css/cool.css:913]: This is not needed we can safely change the colour directly in the jquery-ui-lightness.css`:896`, ideally to `var(--color-background-lighter)`

File: browser/css/cool.css [hunk 2]
```diff
@@ -900,3 +900,54 @@ nav.spreadsheet-color-indicator ~ #sidebar-dock-wrapper {
...
+#datepicker .ui-state-highlight {
+	background: lightgray;
+	font-weight: bold;
+}
+#datepicker .ui-state-active {
+	border: 1px solid #a9a9a9;
+}
+#datepicker .ui-widget.ui-widget-content {
+	border: 1px solid #cccace;
+}
```
[the remark this answers is not in the document]
Comment (pedropintosilva) [reply browser/css/cool.css:926]: probably there are more cases like these two I have pointed out

**Comment by Rash419** [comment]
@pedropintosilva I think we should only edit `.ui-datepicker` rules in jquery-ligthness.css . If we edit for example `ui-state-active`
directly from jquery-lightness.css it will affect all the widgets , ui-state-active's child elements have some specific rules for example
`.ui-state-active a` we also need to change that then

File: browser/css/cool.css [hunk 3]
```diff
@@ -900,3 +900,51 @@ nav.spreadsheet-color-indicator ~ #sidebar-dock-wrapper {
...
+
+#datepicker .ui-widget-content {
+	background: #fff;
+}
```
Comment (pedropintosilva) [inline browser/css/cool.css:925]: This is not needed we can safely change the colour directly in the jquery-ui-lightness.css`:896`, ideally to `var(--color-background-lighter)`

**Review by pedropintosilva — CHANGES_REQUESTED** [review]

**Pull request closed without merging**